src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -194,21 +204,23 @@ void TimerTick(object sender, EventArgs e)
 
         async Task TimerTick()
         {
-            Debug.Assert(client != null, "TimerTick should not be triggered when there is no IMetricsService");
+            Initialize();
 
-            // Subsequent timer ticks should occur every 8 hours.
-            timer.Interval = TimeSpan.FromHours(8);
+            if (firstRun)
+            {
+                IncrementLaunchCount();
+                timer.Interval = TimeSpan.FromHours(8);
+                firstRun = false;
+            }
 
             if (userSettings == null)
```

Comment:
The `userSettings` check is unnecessary, you're right. However we still need to stop the timer if `client == null`. I think you added the `SwitchToMainThreadAsync` there - is that still necessary? Not sure why it's needed.